components/AvalancheForecastZoneMap.tsx

```diff
@@ -35,39 +45,58 @@ export interface MapProps {
 
 export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({center, date}: MapProps) => {
   const [isReady, setIsReady] = useState<boolean>(false);
-  const [region, setRegionBounds] = useState<RegionBounds>({
-    topLeft: {latitude: 0, longitude: 0},
-    bottomRight: {latitude: 0, longitude: 0},
-  });
 
   function setReady() {
     setIsReady(true);
   }
 
-  const largerRegion: Region = regionFromBounds(region);
-  largerRegion.latitudeDelta *= 1.05;
-  largerRegion.longitudeDelta *= 1.05;
   const {isLoading, isError, data: zones} = useMapViewZones(center, date);
 
   const [selectedZone, setSelectedZone] = useState<MapViewZone | null>(null);
   const onPress = useCallback(() => {
     setSelectedZone(null);
   }, []);
 
+  const regionBounds: RegionBounds = zones
+    ? zones.reduce((accumulator, currentValue) => updateBoundsToContain(accumulator, toLatLngList(currentValue.geometry)), defaultRegionBounds)
+    : defaultRegionBounds;
+  zones?.forEach(zone => updateBoundsToContain(regionBounds, toLatLngList(zone.geometry)));
+  const region: Region = regionFromBounds(regionBounds);
```

Comment:
`region` and `regionBounds` are kind of generic, is there a more specific name that isn't horrible to type? something like `avalancheCenterRegion` but maybe shorter?